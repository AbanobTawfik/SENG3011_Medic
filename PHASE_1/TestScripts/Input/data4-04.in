#!/bin/sh
# searching for a city with one associated report 1

start_date=2014-01-01T00:00:00
end_date=2014-12-30T23:59:59
location=Rome
