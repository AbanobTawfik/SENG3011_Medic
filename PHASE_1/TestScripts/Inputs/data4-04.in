#!/bin/sh

start_date=2014-01-01T00:00:00
date_end=2014-12-30T23:59:59
location=Rome
