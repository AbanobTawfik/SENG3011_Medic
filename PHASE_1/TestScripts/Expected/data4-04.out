{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "articles": [
        {
            "url": "url16",
            "reports": [
                {
                    "diseases": [],
                    "syndromes": [],
                    "event_date": "2014-01-01 xx:xx:xx to 2014-12-30 xx:xx:xx",
                    "locations": [
                        {
                            "country": "Italy",
                            "location": "Rome",
                            "geonames_id": 12345
                        },
                        {
                            "country": "Iran",
                            "location": "Tehran",
                            "geonames_id": 23456
                        }
                    ]
                }
            ],
            "headline": "Headline of url16",
            "main_text": "This is the main text for url16.",
            "date_of_publication": "2014-12-31 12:00:00"
        }
    ]
}
